blackduck/Client.py

```diff
@@ -102,7 +100,7 @@ def __init__(
         """
         self.base_url = base_url
         self.session = session or HubSession(base_url, timeout, retries, verify)
-        self.session.auth = (auth or BearerAuth)(session=self.session, token=token, username=username, password=password)
+        self.session.auth = auth or BearerAuth(self.session, token)
```

Comment:
@gsnyder2007 do we want to outright prevent cookie authentication on this client? I was of the opinion that we should inform users that they shouldn't be doing it this way (through docs & logs) but not outright prevent it as there may be some viable uses.